main.go

```diff
@@ -506,19 +519,25 @@ func updateScreenshotNames(testLogsDir string) error {
 	} else if len(testSummariesPths) == 0 {
 		return fmt.Errorf("no TestSummaries file found with pattern: %s", testSummariesPattern)
 	}
-	testSummariesPth := testSummariesPths[0]
 
-	testSummariesContent, err := fileutil.ReadStringFromFile(testSummariesPth)
+	//
+	// TestSummaries
+	testSummariesPth := testSummariesPths[0]
+	testSummaries, err := xcodeutil.New(testSummariesPth)
```

Comment:
xcodeutil.New sound as it would create a new xcodeutil, pls rename it: NewTestSummaries